### PR TITLE
chore(docs): Tracetest Cloud x K6 Recipe

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the Tracetest Cloud x K6 recipe to the docs

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Screeenshots

localhost_3000_examples-tutorials_recipes_running-performance-tests-tracetest-cloud-with-k6

### CODE REVIEW

File: docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+[K6](https://k6.io/) is an open-source performance testing tool used for testing the performance of APIs, microservices, and websites. It is designed to be developer-centric, making it easy for developers to write and maintain performance tests as code.
+
+## Performance Tests and Trace-Based Testing using Tracetest Cloud and K6 (Pokeshop Demo)
+
+This is a simple quick start guide on how to run the Tracetest K6 extension to run enhanced performance tests with Trace-Based testing. The infrastructure will use the Pokeshop Demo as a testing ground, triggering requests against it and generating telemetry data.
+
+## Prerequisites
+
+- Sign up to [app.tracetest.io](https://app.tracetest.io).
+- Creating an [environment](/concepts/environments).
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx:34]:
```suggestion
- Create an [environment](/concepts/environments).
```

File: docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+- Having access to the environment's [agent token](/configuration/agent).
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx:35]:
```suggestion
- Have access to the environment's [agent token](/configuration/agent).
```

File: docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+### Docker Compose Network
+
+All `services` in the `docker-compose.yaml` are on the same network and will be reachable by hostname from within other services. For example, `jaeger:14250` in the `collector.config.yaml` file will map to the `jaeger` service, where port `14250` is the port where the Jaeger all-in-one instance accepts telemetry data.
+
+## Pokeshop Demo App
+
+The Pokeshop Demo App is a complete example of a distributed application using different backend and front-end services, implementation code is written in Typescript.
+
+:::note
+To find out more about the Pokeshop Demo visit the [documentation page](/live-examples/pokeshop/overview)
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx:55]:
```suggestion
To find out more about the Pokeshop Demo visit the [documentation page](/live-examples/pokeshop/overview).
```

File: docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+      TRACETEST_API_KEY: ${TRACETEST_API_KEY}
+```
+
+Do not forget to set the `TRACETEST_API_KEY` environment variable in the `.env` file to your Tracetest Agent API key available on the settings page of your environment.
+
+The `tracetest.config.yaml` file contains the basic setup of connecting Tracetest to the Postgres instance.
+
+## Building the Tracetest xk6 Binary
+
+The Tracetest extension can be found in the official k6 documentation [here](https://k6.io/docs/extensions/get-started/bundle/)
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-load-tests-tracetest-cloud-with-k6.mdx:172]:
```suggestion
The Tracetest extension can be found in the official k6 documentation [here](https://k6.io/docs/extensions/get-started/bundle/).
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Review by adnanrahic — COMMENTED** [review]
This is really nice!

I have some minor suggestions we can talk about:

1. Naming: We should call the cloud version just "Tracetest" and the OSS is "Tracetest Core". I think the official name for "k6" is with a lowercase "k" always.
2. Where to put this recipe: The other k6 guide we have is under "Tools & Integrations". WDYT about putting this one there as well? We can rename the old one to "k6 & Tracetest Core" and this new one can be "k6 & Tracetest".

Let me know what you think and I can help with the structuring to not waste your time.

**Comment by adnanrahic** [comment]
screencapture-localhost-3000-tools-and-integrations-k6-2023-12-22-10_46_30

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**